x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -154,6 +196,19 @@ export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteIniti
     })
   );
 
+  /**
+   * @api {post} /api/ml/data_frame/_explain Provides additional info for requested analytics job
```

Comment:
The description of this `_explain` endpoint needs enhancing.

Reply:
https://github.com/elastic/kibana/pull/56006/commits/fc9297f3498431f41f91440b5c9ffa664401a3c5